exporter/awsemfexporter/util_test.go

```diff
@@ -382,11 +382,12 @@ func TestProcessAttributes(t *testing.T) {
 			resourceAttributes: map[string]any{
 				keyAttributeEntityServiceName:           "my-service",
 				keyAttributeEntityDeploymentEnvironment: "my-environment",
-				keyAttributeEntityAwsAccountId:          "0123456789012",
+				keyAttributeEntityAwsAccountID:          "0123456789012",
 			},
 			wantedAttributes: map[string]*string{
 				serviceName:           aws.String("my-service"),
 				deploymentEnvironment: aws.String("my-environment"),
+				awsAccountID:          aws.String("0123456789012"),
```

Comment:
This should have been in the previous PR, as it was failing a unit test: https://github.com/amazon-contributing/opentelemetry-collector-contrib/actions/runs/11164502627/job/31034127738